### DISCUSSION TITLE
custom_config.h with gaming layer

### DISCUSSION THREAD

**Discussion by mnivoliez** [manna-harbour/miryoku#354] [Show and tell]
## Summary
- Miryoku Implementations
  - [ ] Miryoku QMK
  - [x] Miryoku ZMK
  - [ ] Miryoku KMonad
  - [ ] Miryoku KMK
- Features
  - Layers
    - [ ] Substitute Layer
    - [x] Add Layer
    - [ ] Remove Layer
    - [ ] Change Layer Display Name
  - Change Mapping
    - [ ] All Layers
    - [x] Per Layer

I added two layer for gaming: alpha on one and num on the other. Good enough for Helldivers 2.
Accessing it through the left outer pinky with a double tap.

**Comment by Stianhn** [comment]
Thanks for config :raised_hands: 

The double tap is on the right side though.

**Reply by dull-kiwi** [reply]
Where in the config can you actually see where the double tap button to activate the layer is? I haven't been able to activate it and want to confirm I am hitting the right key.

**Comment by 22ispencer** [comment]
thanks for the config! figured i'd [link my repo](https://github.com/22ispencer/qmk_userspace/blob/060ae59785f063defceeb2ca7cf0a4bb62871ea4/users/miryoku/custom_config.h) where i modified this to work on QMK

**Reply by 22ispencer** [reply]
at the moment this only works for crkbd 3x6_3
